--- Makefile~	2009-12-02 16:17:20.000000000 +0100
+++ Makefile	2009-12-20 17:49:47.000000000 +0100
@@ -12,7 +12,7 @@
 LIBS=-lm -lz -L/usr/X11R6/lib -lX11 -lXext -lXt -lXrender -lXft -lpng -lavformat -lavcodec -lavutil -lasound -lpthread
 
 # if libswscale is not in libavcodec, add a -lswscale to the LIBS
-LIBS+=`[ -f /usr/lib/libswscale.so -o -f /usr/local/lib/libswscale.so ] && echo "-lswscale"`
+LIBS+=`[ -f /usr/lib/libswscale.so -o -f /usr/local/lib/libswscale.so -o -f /usr/lib64/libswscale.so ] && echo "-lswscale"`
 
 CLASSES=ActionClass.o	\
 	ApplicationClass.o	\
